test/Microsoft.Azure.Devices.Edge.Test/SetupFixture.cs

```diff
@@ -96,6 +117,24 @@ public async Task BeforeAllAsync()
                     {
                         await device.MaybeDeleteIdentityAsync(token);
                     }
+
+                    // Remove packages installed by this run.
+                    await this.daemon.UninstallAsync(token);
+
+                    // Delete test certs, keys, etc.
+                    Directory.Delete("/etc/aziot/e2e_tests", true);
+
+                    // Restore backed up config files.
+                    foreach (string file in this.configFiles)
```

Comment:
@gordonwang0, I noticed that this piece of code has been removed from master but still in release/1.2. Can you please check that this is alright?